src/database/types.tsx

```diff
@@ -0,0 +1,24 @@
+import * as firebase from "firebase/app";
```

Comment:
Should we start trying to define/move model types in the `database/` directory? I think it can be confusing sometimes having the `types` file in `page-multi-facility/` directory when we end up importing those types to other pages (and contexts). I was wondering if it would make sense to have a directory at the same level as `database/` that's caleld `types/` and has the definitions for the model-like types like `Facility`, `Scenario`, etc.

Reply:
I agree with you @daschi.  It's weird that the `types` file still lives in the `page-multi-facility` directory.  I think having a `types` directory at the root level with separate files for our domain types (i.e. `Scenario`, `Facility`, etc.), database types, model calculation types, etc. makes sense.

Reply:
interesting ... I definitely wasn't thinking of`database/` as the place where those would eventually live, this was just for things that are mainly internal to these database functions. 

I'm in favor of this idea though ... there are some things (type definitions, possibly also some helper functions and things like that as well) that pertain to the core data model and don't belong to any single component but are kind of scattered throughout the codebase (e.g. I might include some of the more generic stuff in `infection-model/` also). I don't really want to roll that into this PR but I think it's a good tech debt task for us to return to later if you want to make a ticket @daschi ?

Reply:
sure! I can add that as a ticket